Deployment/VMware vSphere/High Availability/variables.tf

```diff
@@ -5,58 +5,49 @@
 variable "vsphere-user" {
   type        = string
   description = "VMWare vSphere cluster admin user"
-  default     = "<vSphere admin-level user>"
 }
 
 variable "vsphere-password" {
   type        = string
   description = "VMWare vSphere admin user password"
-  default     = "<password>"
 # Turn on for production use. Will hide the entire custom variable entry
 #  sensitive   = true
 }
 
 variable "vsphere-server" {
   type        = string
   description = "VMWare vCenter FQDN or IP address"
-  default     = "<vCenter address>"
 }
 
 variable "vsphere-datacenter" {
   type        = string
   description = "VMWare vSphere Datacenter"
-  default     = "<Datacenter>"
 }
 variable "vsphere-resource-pool" {
   type        = string
   description = "VMWare Resource Pool"
-  default     = "<Select a resource pool>"
 }
 
 # Not currently used for the examples
 variable "vsphere-cluster" {
   type        = string
   description = "VMWare vSphere cluster."
-  default     = "<Cluster name>"
 }
 
 variable "vsphere-host" {
   type        = string
   description = "VMWare host"
```

Comment:
Maybe an issue for a different time, but it seems like there should be either a `vsphere-cluster` or a `vsphere-host`, one or the other.

Reply:
That does step a little far outside the goal of this PR, but I did create an issue to track that.
https://github.com/hammer-space/terraform/issues/2